runbotics/runbotics-scheduler/src/migrations/1730107464046-ProcessInstance.ts

```diff
@@ -0,0 +1,100 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class ProcessInstance1730107464046 implements MigrationInterface {
+    name = 'ProcessInstance1730107464046'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "process_instance" DROP CONSTRAINT "fk_process_instance_user_id"`);
+        await queryRunner.query(`ALTER TABLE "process_instance" DROP CONSTRAINT "fk_process_instance_process_id"`);
+        await queryRunner.query(`ALTER TABLE "process_instance" DROP CONSTRAINT "fk_process_instance_bot_id"`);
+        await queryRunner.query(`ALTER TABLE "process_instance" DROP CONSTRAINT "fk_process_instance_trigger_event"`);
+        await queryRunner.query(`ALTER TABLE "process_instance" DROP CONSTRAINT "fk_process_instance_root_process_instance_id"`);
+        await queryRunner.query(`ALTER TABLE "process_instance_loop_event" DROP CONSTRAINT "fk_process_instance_loop_event_process_instance_id"`);
+        await queryRunner.query(`ALTER TABLE "process_instance_event" DROP CONSTRAINT "fk_process_instance_event_process_instance_id"`);
+        await queryRunner.query(`DROP INDEX "public"."process_instance_process_id_idx"`);
+        await queryRunner.query(`DROP INDEX "public"."process_instance_root_process_instance_id_idx"`);
+        await queryRunner.query(`DROP INDEX "public"."process_instance_loop_execution_id_idx"`);
+        await queryRunner.query(`DROP INDEX "public"."process_instance_event_process_instance_id_idx"`);
```

Comment:
They may be useful (there are quite a lot of process instance events, so a few Indexes sorted by e.g. process instance or event Id can increase the efficiency greately)

Reply:
You can create new ones